ops/ccs-ops-misc/load_test/README.md

```diff
@@ -70,24 +70,34 @@ The tests are run with parameters that specify all the test params and test file
+    python3 runtests.py --homePath="<home_directory_path>" --clientCertPath="<home_directory_path>/bluebutton-backend-test-data-server-client-test-keypair.pem" --databaseUri="postgres://<db-username>:<db-password>@<db-host>:<port>/<db-name>" --testHost="https://test.bfd.cms.gov" --testFile="./v2/eob_test_id_count.py" --testRunTime="1m" --maxClients="100" --clientsPerSecond="5"
+
+If you have existing configuration in `./config.yml`, you can also run the tests via:
+
+```
+python3 runtests.py --testFile="<your-test-file>"
+```
+
+Or, if you have some YAML configuration in a different file (note that these values will be saved to the root `./config.yml`, so subsequent runs can omit the `configPath` if you are not changing anything):
+
+```
+python3 runtests.py --configPath="config/<your-config-here>.yml" --testFile="<your-test-file>" <other-cli-args-here>...
+```
 
 Essentially, all the items you would set up in the config file are set in a single line. There are some optional, and some required parameters here:
 
 **--homePath** : (Required) : The path to your home directory on the local box, such as /home/logan.mitchell/
 
 **--clientCertPath** : (Required) : The path to the PEM file we copied/modified earlier. Should be located in your home directory like this: ~/bluebutton-backend-test-data-server-client-test-keypair.pem
 
-**--databaseHost** : (Required) : The database host, used to pull data. Get this from the database endpoint we noted in the earlier step "Getting The Database Instance"
-
-**--databaseUsername** : (Required) : The username for connecting to the database, can be found in Keybase. Make sure to use the correct one for the environment you're connecting to.
-
-**--databasePassword** : (Required) : The password for connecting to the database, can be found in Keybase. Make sure to use the correct one for the environment you're connecting to.
+**--databaseUri** : (Required) : The URI used for connecting to the database server. Needs to include username, password (both can be found in Keybase, make sure to use the correct one for the environment you're connecting to), hostname and port.
 
 **--testHost** : (Required) : The load balancer or single node to run the tests against. The environment used here should match the same environment (test, prod-sbx, prod) as the database, so we pull the appropriate data for the environment tested against. Note that when using a single node, you should specity the Ip AND the port for the application.
 
 **--testFile** : (Required) : The path to the test file we want to run.
 
+**--configPath** : (Optional) : The path to a YAML configuration file that will be read from (but not _written_ to) for the values specified here. The values in this configuration file will be merged with values from the CLI, with the CLI values taking priority. The resulting merged values will then be written to the root `config.yml` file. If not provided, defaults to `config.yml` (the root YAML configuration file). 
```

Comment:
A little specific but it may be written to, if you use the root path or use the default, right? We read from this which may also be the place we write to. Not sure if we can reword this to be more specific, or if its worth doing so, since its relatively clear as is.

Reply:
Yep, you're correct. If `configPath` is `./config.yml` (the default config path, or the root configuration file) we will read _and_ write to it. If the `configPath` is _not_ `./config.yml` (some other YAML file) then we will _read_ from `configPath` and then write to `./config.yml` so that the file at `configPath` isn't overwritten unexpectedly.

I think I'll take a variation of what I wrote above here and rewrite that explanation, because looking at it now it's definitely a bit confusing.